spec/mailers/emailer_spec.rb

```diff
@@ -98,7 +98,8 @@
       mail = Emailer.account_update(@record)
 
       VCR.use_cassette('emailer/account_update') do
-        name = 'Hello Jane Doe,'
+        # name = 'Hello Jane Doe,'
```

Comment:
Why leave the old functionality in comments in all these tests?

Reply:
We'll need them later for the follow up ticket: https://sfgovdt.jira.com/browse/DAH-2471

lmk if you prefer to just remove lines for this kind of situation

Reply:
All good to leave them then! Could you include a todo code comment that references that ticket?

Reply:
I am okay with the comments in this case